npm/react/plugins/craco/index.js

```diff
@@ -2,13 +2,13 @@ const { startDevServer } = require('@cypress/webpack-dev-server')
 const { createWebpackDevConfig } = require('@craco/craco')
 const { getLegacyDevServer } = require('../utils/legacy-setup-dev-server')
 
-function devServer (cypressDevServerConfig, cracoConfig, indexHtml) {
+function devServer (cypressDevServerConfig, cracoConfig, indexHtmlFile) {
```

Comment:
This will be a breaking change if someone is using `indexHtml`, won't it?

Reply:
`indexHtml` was only added on `10.0-release` so this update would not be breaking since it was never released.

Reply:
Oh, I thought this was in develop. All good.